.github/workflows/aws.yml

```diff
@@ -6,6 +6,9 @@ jobs:
   deploy:
     name: Build and deploy lambda
     runs-on: ubuntu-latest
+     strategy:
+      matrix:
+        payment-platform: ['stripe', 'fastspring']
```

Comment:
did this just as we do in angular portals
may not be right

Reply:
The build and deploy scripts are for reference, they differ for most customers, these will need to be separate for every integration or if they are similar, might need some sort of input from `dispatch`.

Reply:
@azan-n for now I have used dispatch.